### PR TITLE
Version

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/trading-backend/pull/61
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/1068

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 *It is strongly advised to perform an update of your tentacles after updating OctoBot. (start.py tentacles --install --all)*
 
+## [1.0.11] - 2024-07-05
```
Comment (Herklos) [inline CHANGELOG.md:9]: Should we make a special version for these new features like 1.1.0 or 2.0.0?

Comment (GuillaumeDSM) [reply CHANGELOG.md:9]: Good idea, let's go for 1.1.0 !

Comment (GuillaumeDSM) [reply CHANGELOG.md]: up

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 *It is strongly advised to perform an update of your tentacles after updating OctoBot. (start.py tentacles --install --all)*
 
+## [2.0.0] - 2024-07-05
```
Comment (Herklos) [inline CHANGELOG.md:9]: 🎉

**Review by Herklos — APPROVED** [review]
💯

**Pull request merged**